FrenchFilter/sections/antiadblock.txt

```diff
@@ -26,6 +26,9 @@ hollywoodpq.com#%#//scriptlet("set-constant", "ujloijdkhjkwus", "false")
 monacomatin.mc#@##adsense
 monacomatin.mc#@#.AdSense
 monacomatin.mc#%#//scriptlet("abort-current-inline-script", "$", "checkAds")
+! https://github.com/AdguardTeam/AdguardFilters/issues/65697
+topnewsshow.com#@#.an-sponsored
+topnewsshow.com#@#.an-advert-banner
```

Comment:
I don't see the adblock detection. Do you still have it?

Reply:
I've reproduce it, It is caused by anOptions script, please add to the appropriate section.

Reply:
Hm, in this case remove those rules and add the domain to the `anOptions` section in the Base filter. If you can close the message, then only add it to the Annoyances.